pom.xml

```diff
@@ -460,6 +462,36 @@
           </configuration>
         </plugin>
 
+        <plugin>
+          <groupId>org.moditect</groupId>
+          <artifactId>moditect-maven-plugin</artifactId>
+          <version>${moditect-maven-plugin.version}</version>
+          <executions>
+            <execution>
+              <id>add-module-infos</id>
+              <phase>package</phase>
+              <goals>
+                <goal>add-module-info</goal>
+              </goals>
+              <configuration>
+                <skip>${moditect.skip}</skip>
+                <overwriteExistingFiles>true</overwriteExistingFiles>
+                <module>
+                  <moduleInfo>
+                    <!-- module name will be derived from filename -->
+                    <!-- export everything -->
+                    <exports>*;</exports>
+                    <!-- declare services consumed by the artifact -->
+                    <addServiceUses>true</addServiceUses>
+                  </moduleInfo>
+                </module>
+                <jdepsExtraArgs>
+                  <arg>--multi-release=9</arg>
+                </jdepsExtraArgs>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
```

Comment:
can we move this whole section to line 533 (outside `pluginManagement`), I see no reason to keep it in here

Reply:
Yes, it can be safely moved.

Reply:
Yeah. I moved, push and merged... didn't realized the push got a conflict and didn't moved 🤦‍♂️